src/digest.rs

```diff
@@ -43,38 +44,39 @@ pub trait DigestToF: Digest {
                 .map_err(|e| io::Error::new(io::ErrorKind::InvalidData, e))?,
         );
 
-        Ok(into_field_by_bits(digest.deref(), NUM_HASH_BITS))
+        Ok(onto_curve_by_bits(digest.deref(), NUM_HASH_BITS))
     }
 }
-impl DigestToF for sha3::Sha3_256 {}
+impl DigestToCurve for sha3::Sha3_256 {}
 
-fn into_field_by_bits<F: PrimeField>(input: &[u8], bits_count: NonZeroUsize) -> F {
-    let mut coeff = F::ONE;
+fn onto_curve_by_bits<C: CurveAffine>(input: &[u8], bits_count: NonZeroUsize) -> C {
```

Comment:
typo: `into`

Reply:
I thought it was onto the curve. changed to "into"